src/ui/page/dataverse/dataspace/governance/governanceContent.tsx

```diff
@@ -0,0 +1,61 @@
+import type { FC } from 'react'
+import { useParams } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import { BackButton } from '@/ui/view/dataverse/component/backButton/backButton'
+import { GovernanceNavigation } from './governanceNavigation'
+import type { SectionDTO, SubSectionDTO } from './mockedData'
+import './governance.scss'
+
+type SectionProps = {
+  section: SectionDTO
+}
+
+export const Section: FC<SectionProps> = ({ section }) => {
+  const { subsectionId } = useParams<{ subsectionId: string }>()
+  const subsection =
+    section.contains.find(subSection => subSection.id === subsectionId) ?? section.contains[0]
```

Comment:
Please , add memoization

Reply:
This component will be removed in https://github.com/okp4/dataverse-portal/pull/185.